scripts/build.ps1

```diff
@@ -232,6 +232,22 @@ function CMake-Build {
         if (!$DisableTools) {
             Copy-Item (Join-Path $BuildDir "obj" $Config "msquicetw.lib") $ArtifactsDir
         }
+        if ($PGO -and $Config -eq "Release") {
+            # TODO - Figure out a better way to get the path?
+            $VCToolsPath = "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.26.28801\bin\Host$Arch\$Arch"
```

Comment:
Apparently these paths don't exist on AZP machines. @ThadHouse any ideas where they'd be?

Reply:
AZP uses enterprise VS. So `C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise` instead of Buildtools